jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/ToolkitAddConnectionDialog.kt

```diff
@@ -32,8 +32,8 @@ import software.aws.toolkits.core.utils.warn
 import software.aws.toolkits.jetbrains.ToolkitPlaces
 import software.aws.toolkits.jetbrains.core.coroutines.getCoroutineUiContext
 import software.aws.toolkits.jetbrains.core.coroutines.projectCoroutineScope
-import software.aws.toolkits.jetbrains.core.credentials.sono.ALL_SONO_SCOPES
 import software.aws.toolkits.jetbrains.core.credentials.sono.ALL_SSO_SCOPES
+import software.aws.toolkits.jetbrains.core.credentials.sono.CODEWHISPERER_SCOPES
```

Comment:
now since we are touching this:
Do we want to rename "sono" to it's current name?

Reply:
it's actually ok to address the rename in a different issue if we want.

Reply:
there are a lot of internal references, so let's tackle that separately if we want to do that